### PR TITLE
My first commit

### PR DESCRIPTION
**Pull request by Hadiqamrana**

### CODE REVIEW

**Comment by codewithmide** [comment]
Hi @Hadiqamrana, well done on making your first contribution. However, it noticed that your README file does not follow the markdown linting style.

If you use VSCode as your preferred code editor, I'll advise you to install this extension [markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) to help you fix it

**Comment by Hadiqamrana** [comment]
@codewithmide thanks for the review surely will fix that !

**Comment by codewithmide** [comment]
You're welcome

File: 2023/Hadiq-Amran/README.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+Note : These useful resources I found i believe are of importance to any one who wants to understand Wagtail as project, especially some who has that same tests as me in Django 
+In summary, these links collectively provide a roadmap for both beginners and experienced developers interested in understanding, contributing to, or working with Wagtail. They cover essential aspects such as contributing guidelines, formatting conventions, specific project opportunities, user interface guidance, and comprehensive official documentation, making them indispensable resources for anyone involved with Wagtail.
+I hope they are helpful to some one !
+
+
+#Added Recent Learning Adventures Blog Post 
+Hello, Everyone i made a blog where i talk about my new learning adventures and the cool things I'm learning.I'm really excited to share this journey with all of you. you can checkout my blog at https://medium.com/@amranahadeeq/learning-with-wagtail-a-fun-adventure-faef340b4ecd
+
+#Added A Django Blog post 
+You can check out this blog post at https://medium.com/@amranahadeeq/django-streamlining-and-accelerating-web-development-e23c82f9c4c6
```
Comment (thibaudcolas) [inline 2023/Hadiq-Amran/README.md:119]: This is also very generic, I see nothing in there about your own experience.

**Pull request merged**